collect_app/src/main/java/org/odk/collect/android/activities/AppListActivity.java

```diff
@@ -327,7 +328,15 @@ public void onItemClicked(SortDialogAdapter.ViewHolder holder, int position) {
         }
     }
 
-    protected void showSnackbar(String result) {
+    protected void showSnackbar(@NonNull String result) {
         SnackbarUtils.showSnackbar(llParent, result);
     }
+
+    protected void setToolbarProgress(boolean visibility) {
+        progressBar.setVisibility(visibility ? View.VISIBLE : View.GONE);
+    }
+
+    protected boolean isToolbarProgressVisible() {
```

Comment:
`isProgressBarVisibile`? I didn’t look to see if there are more than one ProgressBars.